groups/infrastructure/profiles/development-eu-west-2/cidev/vars

```diff
@@ -0,0 +1,9 @@
+environment = "cidev"
+aws_profile = "development-eu-west-2"
+enable_container_insights = true
+
+ec2_key_pair_name = "chs-cidev"
+
+# Scheduled scaling of tasks
+service_scaledown_schedule = "55 19 * * ? *"
+service_scaleup_schedule   = "5 6 * * ? *"
```

Comment:
You're not defining any capacity for the ASG - the applicable vars default to `0` and so vars should be added here to set the ASG capacity appropriately.